### PR TITLE
Added missing StarterPack for MvvmCross.Forms #2073 

### PR DESCRIPTION
**Pull request by brucewilkins**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
Missing StarterPack for MvvmCross.Forms #2073 

### :arrow_heading_down: What is the current behavior?
No starter pack

### :new: What is the new behavior (if this is a feature change)?
Starter pack added

### :boom: Does this PR introduce a breaking change?
No

### :bug: Recommendations for testing
If using the default Xamarin Forms template, ensure the PCL is targeting a profile compatible with the MvvmCross.Forms dependency i.e.: Profile111.

### :memo: Links to relevant issues/docs
https://github.com/MvvmCross/MvvmCross/issues/2073

### :thinking: Checklist before submitting

- [ ] All projects build
- [ ] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [ ] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contribute/mvvmcross-docs-style-guide))
- [x] Nuspec files were updated (when applicable)
- [ ] Rebased onto current develop

### CODE REVIEW

File: nuspec/Forms/CoreContent/MvxFormsViewModel.cs.pp [hunk 1]
```diff
@@ -0,0 +1,32 @@
+using System.Threading.Tasks;
+using MvvmCross.Core.ViewModels;
+
+namespace $rootnamespace$.Core.ViewModels
+{
+    public class MvxFormsViewModel : MvxViewModel
```
Comment (willsb) [inline nuspec/Forms/CoreContent/MvxFormsViewModel.cs.pp]: Wouldn't it be better to name this `MainViewModel` for consistency with the other StarterPacks?

File: nuspec/Forms/CoreContent/MvxFormsViewModel.cs.pp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            get { return _text; }
+            set { SetProperty(ref _text, value); }
+        }
+
+        public override Task Initialize()
+        {		    
+            return base.Initialize();
+        }        
+        
+        private void ShowText()
```
Comment (willsb) [inline nuspec/Forms/CoreContent/MvxFormsViewModel.cs.pp]: This method's name is misleading

File: nuspec/Forms/CoreContent/MvxFormsPage.xaml.cs.pp [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using MvvmCross.Forms.Core;
+using $rootnamespace$.Core.ViewModels;
+using Xamarin.Forms;
+
+namespace $rootnamespace$.Core.Pages
+{
+    public partial class MvxFormsPage : MvxContentPage<MvxFormsViewModel>
```
Comment (willsb) [inline nuspec/Forms/CoreContent/MvxFormsPage.xaml.cs.pp]: Similar to the ViewModel, I'd name this `MainPage`

**Comment by martijn00** [comment]
Let's merge this in and make an issue with the missing things and improvements and make that up-for-grabs.

**Pull request merged**